collector-http/src/main/java/zipkin2/collector/otel/http/DefaultOtelResourceMapper.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package zipkin2.collector.otel.http;
+
+import io.opentelemetry.proto.resource.v1.Resource;
+import io.opentelemetry.semconv.ServiceAttributes;
+import zipkin2.Span;
+
+/**
+ * Default implementation of {@link OtelResourceMapper} that simply maps resource attributes except for {@link ServiceAttributes#SERVICE_NAME} to tags with optional prefix.
+ */
+public class DefaultOtelResourceMapper implements OtelResourceMapper {
+  private final String resourceAttributePrefix;
+
+  private static final String DEFAULT_PREFIX = "";
+
+  public DefaultOtelResourceMapper(String resourceAttributePrefix) {
```

Comment:
avoid public ctors for builders.

Reply:
fixed in https://github.com/openzipkin-contrib/zipkin-otel/pull/19/commits/efdbae743ce4ea3031ead46255028222e47120ca